MastodonSDK/Sources/MastodonCore/Service/API/APIService+Block.swift

```diff
@@ -22,6 +22,36 @@ extension APIService {
         let isFollowing: Bool
     }
     
+    @discardableResult
+    public func getBlocked(
+        authenticationBox: MastodonAuthenticationBox
+    ) async throws -> Mastodon.Response.Content<[Mastodon.Entity.Account]> {
+        let managedObjectContext = backgroundManagedObjectContext
+        
+        let response = try await Mastodon.API.Account.blocks(
+            session: session,
+            domain: authenticationBox.domain,
+            authorization: authenticationBox.userAuthorization
+        ).singleOutput()
+        
+        let userIDs = response.value.map { $0.id }
+        let predicate = NSPredicate(format: "%K IN %@", #keyPath(MastodonUser.id), userIDs)
```

Comment:
Maybe missing This.
Same to https://github.com/mastodon/mastodon-ios/pull/686#discussion_r1031330565

Reply:
I also noticed it, I fixed this with my latest commit. Thx 👍